ops/terraform/modules/resources/eft_efs/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "partner" {
+  description = "Parnert name. E.g., 'bcda', 'dpc', etc."
+  type        = string
+}
+
+variable "partner_acct_num" {
+  description = "Partners AWS Account Number"
+  type        = string
+}
+
+variable "partner_subnets" {
+  description = "Partner subnets requiring access to EFT EFS file systems"
+  type        = map(list(string))
```

Comment:
Elsewhere this is defined as `map(map(list(string)))`.  Is this intentional?  Just checking!

Reply:
We store all the partners info in a `map(map(list(string)))` (ie in a .tfvars file):
<img width="260" alt="Screen Shot 2021-05-21 at 12 20 51 PM" src="https://user-images.githubusercontent.com/539260/119175264-3926b780-ba2f-11eb-8f17-fa01b7f675ac.png"> 

But when we are provisioning a specific partner, we only pass their info to the module like so:
<img width="363" alt="Screen Shot 2021-05-21 at 12 15 07 PM" src="https://user-images.githubusercontent.com/539260/119175459-77bc7200-ba2f-11eb-9862-7653bc095ce8.png">

Reply:
Thanks, the module instantiation and tfvars context helps.